### DISCUSSION TITLE
Adding back the files that have changed after restoring from an old backup

### DISCUSSION THREAD

**Discussion by patrick-5546** [nextcloud/all-in-one#1397] [Questions] [answered]
I forgot to backup before updating the containers in v2.1.2 and now my setup is broken. My last backup is about 2 weeks old. Is there any recommended procedure for adding back the files that have changed after restoring from an old backup?

I am on Unraid, would this work?
1. Duplicate my nextcloud data share
2. Restore from backup
3. Point mastercontainer to the duplicated nextcloud data share and restart container
4. Delete original nextcloud data share

But of course fixing my setup without restoring from an old backup would be preferable. This issue describes what is wrong with my setup: https://help.nextcloud.com/t/update-failure-from-23-0-6-1-to-24-0-5-1/145734. Would downgrading to v2.1.1 fix my setup?

Thanks in advance for the help

**Comment by szaimen** [comment]
Sorry I cannot help you if you dont create a backup before updating.

**Comment by szaimen** [comment, marked as answer]
In general you would need to back up the current state, restore from backup, restore the files from the broken state manually and rescan the files with `occ files:scan --all`.
